src/modules/site-v2/base/utils/auth.py

```diff
@@ -172,3 +172,24 @@ def expired_token_callback(_jwt_header, jwt_data):
   resp = make_response(redirect(url_for('auth.refresh')))
   unset_access_cookies(resp)
   return resp, 302
+
+
+
+def access_token_required(token):
+  '''
+    Require a "Bearer" access token in the request.
+  '''
+  def wrapper(fn):
+    @wraps(fn)
+    def decorator(*args, **kwargs):
+
+      # Check for access token in request
+      access_token = request.headers.get('Authorization')
+      if access_token != 'Bearer {}'.format(token):
+        abort(403)
```

Comment:
quick note on semantics -- should this be 401 (authn: user didn't provide valid credentials) vs 403 (authz: user provided valid credentials, but is not authorized to do this action).

Reply:
Good catch, fixed in 25cfcc10b9f530cd5e69dba966edd5f731790d8c